app/app.go

```diff
@@ -323,7 +323,9 @@ func NewOsmosisApp(
 			app.DistrKeeper.SetParams(ctx, distrParams)
 
 			// configure upgrade for gamm module's pool creation fee param add
-			app.GAMMKeeper.SetParams(ctx, gammtypes.NewParams(sdk.Coins{sdk.NewInt64Coin("uosmo", 1)})) // 1 uOSMO
+			gammParams := app.GAMMKeeper.GetParams(ctx)
```

Comment:
Doesn't this need to set some new params as well?

Reply:
Ah this isn't going in v4 so its not needed. We should make an issue for describing whats going to be needed in the `v5` upgrade handler then

Reply:
So the whole set params part isnt needed?

`	// configure upgrade for gamm module's pool creation fee param add
			gammParams := app.GAMMKeeper.GetParams(ctx)
			gammParams.PoolCreationFee = sdk.Coins{sdk.NewInt64Coin("uosmo", 1)} // 1 uOSMO
			app.GAMMKeeper.SetParams(ctx, gammParams)`